misc/python/materialize/cli/run.py

```diff
@@ -305,9 +305,6 @@ def _build(args: argparse.Namespace, extra_programs: list[str] = []) -> int:
     env = dict(os.environ)
     command = _cargo_command(args, "build")
     features = []
-    if args.tokio_console:
-        features += ["tokio-console"]
-        env["RUSTFLAGS"] = env.get("RUSTFLAGS", "") + " --cfg=tokio_unstable"
```

Comment:
Excited to see this go. You missed a spot though!

https://github.com/MaterializeInc/materialize/blob/eea35ba4ae1712159d7d20f36c5b6f81965b01f2/misc/python/materialize/cli/run.py#L103-L107

Reply:
I am leaving this so people don't need to know to pass `--tokio-console-listen-addr=127.0.0.1:6669`, but can remove in followup if need be

Reply:
Oh, I think we should enable that listen address by default. No reason not to, right?

Reply:
It turns off the tracing fast-path optimization as tokio-console traces are `trace!` level, and may use an arbitrary amount of memory, is that considered reasonable as the default?

Reply:
Ah, got it, probably good to leave it disabled by default then.